src/configuration/registrations.ts

```diff
@@ -103,12 +104,24 @@ export function registerDependencies(
     container.register(
         'storageAdapter',
         asFunction(() => {
-            const bucketName = process.env.GCP_BUCKET_NAME || '';
-            return new GCPStorageAdapter(
-                bucketName,
-                deps.globalLogging,
-                process.env.GCP_STORAGE_EMULATOR_HOST ?? undefined,
-            );
+            if (
+                process.env.LOCAL_STORAGE_PATH &&
+                process.env.LOCAL_STORAGE_HOSTING_URL
+            ) {
+                return new LocalStorageAdapter(
+                    process.env.LOCAL_STORAGE_PATH || '',
+                    new URL(process.env.LOCAL_STORAGE_HOSTING_URL || ''),
+                );
+            }
+            if (process.env.GCP_BUCKET_NAME) {
```

Comment:
Hmm - this is a slight functionality change, I think I should remove the `if` statement here